### ISSUE TITLE
Cannot create case: javafx/scene/paint/Color

### ISSUE THREAD

**Issue by cooperfrench95** [sleuthkit/autopsy#5390]
Platform: Ubuntu 18.04

I followed all the installation instructions and Autopsy does start. However when I try to create a case I get this:

Cannot create case: javafx/scene/paint/Color. In an error popup.

Output of `java -version`:

```
openjdk version "1.8.0_232"
OpenJDK Runtime Environment (Zulu 8.42.0.23-CA-linux64) (build 1.8.0_232-b18)
OpenJDK 64-Bit Server VM (Zulu 8.42.0.23-CA-linux64) (build 25.232-b18, mixed mode)
```

```
----~$ env | grep JAVA_HOME
JAVA_HOME=/usr/lib/jvm/zre-8-amd64
```

For real though. Can you make your program easier to use? I tried installing the version that runs in the browser with `sudo apt-get install autopsy`, but that's broken too. When adding an image which 100% has the correct path, I get: `Invalid wild image (img_path) argument` in that version.

**Comment by esaunders** [comment]
Sounds like you are missing Java FX. Do you see jfxrt.jar in /usr/lib/jvm/zre-8-amd64/lib/ext?
If not you can get Java FX here: https://www.azul.com/downloads/zulu-community/?&version=java-8-lts&os=&os=ubuntu&architecture=x86-64-bit&package=jre-fx&show-old-builds=true

Extract it into your JAVA_HOME location. See the Running_Linux_OSX.txt file for the 'tar' command to use.

**Comment by cooperfrench95** [comment]
@esaunders Turns out the problem was not that I didn't have JavaFX installed, but that I had installed 2 versions of java 8 somehow and Autopsy was using the wrong one which didn't have any FX. Uninstalling one of them solved the problem for me. Thankyou!

**Comment by IrregularLogic** [comment]
It's been a while and I dropped trying autopsy until now and had more luck now. But still issues and questions.
(1) Why do I have to install a custom java environment at all? There is one installed on my PC that came with package management and has never caused issues.
(2) I have now been able finally to install autopsy/sleuthkit latest version and successfully ran it on an old 80GB drive to see how it works and nicely it found quite a lot of stuff.
BUT: today I wanted to run it again and it failed. Error is "Cannot create case" - "void org.sleuthkit.datamodel.SleuthkitJNI$CaseDbHandle.<init>(vava.lang.String.orgsleuthikit.datamodel.SleuthikitJNI$1)'

Why is that and how to fix it?
(3) At the very very first run of autopsy the splash screen (that one with the dog) is shown and BEHIND THAT VERY SCREEN DIRECTLY is a form that wants to be clicked, but as it is hidden behind the dog this is not possible. I already thought it does not work, but then I switched windows using KDE window manager and was able to X-click the window (close it whatever it does). Why cant the java windows be opened and shown at a place where one can actually see them an intereact with them?

(4) Running autopsy first time: when it did work I clicked the domains "Images/Videos", "Communications", "Geolocations" etc. to see what these are doing. But the windows that were shown were hidden half way. The windows were opened at top of visible screen area and their top half was moved outside this area so it was totally impossible to see their content nor click there nor drag the window away to a more convenient location.
Why is that and how do I fix it?

Thanks for this tool but some things would like improvement...
